packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -108,6 +108,8 @@ sealed class _DebugSemanticsRoleChecks {
     SemanticsRole.tab => _semanticsTab,
     SemanticsRole.tabBar => _semanticsTabBar,
     SemanticsRole.tabPanel => _noCheckRequired,
+    SemanticsRole.dialog => _noCheckRequired,
```

Comment:
For scannability, might be good to keep these sorted?

Reply:
yup